src/views/Favorite/FavoriteView.vue

```diff
@@ -1,12 +1,37 @@
 <template>
-    <p>Favoritos</p>
+  <section class="w-full justify-between">
+      <div class="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-4 p-4">
+        <!-- <movies-card v-for="movie in favoriteMovies" :key="movie.id" :movie="movie"/> -->
+        <series-card v-for="series in favoriteSeries" :key="series.id" :movie="series"/>
+      </div>
+    </section>
   </template>
-  
-  <script lang="ts">
-  import { defineComponent } from 'vue';
-  
+
+<script lang="ts">
+import { MoviesModel } from '@/model/movies.model';
+import { SeriesModel } from '@/model/series.model';
+import { SaveFavorites } from '@/utils/saveFavorites.util';  
+import { defineComponent } from 'vue';
+
   export default defineComponent({
-    name: 'FavoriteView'
-  });
-  </script>
-  
\ No newline at end of file
+    data() {
+      return{
+        favoriteMovies: [] as MoviesModel[],
+        favoriteSeries: [] as SeriesModel[],
+        saveFavorites: new SaveFavorites()
+      };      
+    },
+    mounted() {
+        this.getFavorites();
+    },
+    methods:{
+      getFavorites(){
+        const saveFavorites = new SaveFavorites();
+        const allFavorites = saveFavorites.favorites;
+        this.favoriteMovies = allFavorites.filter((item): item is MoviesModel => 'original_title' in item);
```

Comment:
You should have implemented this filtering logic when set item on storage. Something like key type : serie or movie.